src/main/java/core/basesyntax/suppliers/FigureSupplier.java

```diff
@@ -0,0 +1,35 @@
+package core.basesyntax.suppliers;
+
+import core.basesyntax.Color;
+import core.basesyntax.figures.Circle;
+import core.basesyntax.figures.Figure;
+import core.basesyntax.figures.IsoscelesTrapezoid;
+import core.basesyntax.figures.Rectangle;
+import core.basesyntax.figures.RightTriangle;
+import core.basesyntax.figures.Square;
+
+public class FigureSupplier {
+
+    private static final double RANDOM_BOUND = 100.00;
+
+    public Figure getRandomFigure() {
+
+        ColorSupplier colorSupplier = new ColorSupplier();
+        Color randomColor = colorSupplier.getRandomColor();
+        double randomNumber = ColorSupplier.random.nextDouble(RANDOM_BOUND);
```

Comment:
It's better to use switch-case here

```
int index = random.nextInt(NUMBER_OF_FIGURE);
switch (index) {
            case 0:
                return new Circle(randomColor, randomSize());
```

Where
```
private double randomSize() {
        return random.nextDouble() * MAXIMUM_VALUE;
    }
```

Reply:
not fixed